src/lib/storage/dictionary_column.hpp

```diff
@@ -31,7 +31,11 @@ class DictionaryColumn : public BaseDictionaryColumn {
   // return the value at a certain position. If you want to write efficient operators, back off!
   const AllTypeVariant operator[](const ChunkOffset chunk_offset) const override;
 
+  // Returns whether a value is NULL
+  bool is_null(const ChunkOffset chunk_offset) const;
```

Comment:
it seems like this could cause some confusion with out global `is_null`. Maybe the global one should be renamed instead of polluting the namespace with a rather basic name.

Reply:
Good point. Should we rename the global one to `variant_is_null`?

Reply:
Yes, that seems like a reasonable name.

Reply:
done